dl_fanfiction_net.py

```diff
@@ -40,12 +41,25 @@ def print_soup(soup):
     for paragraph in paragraphs:
         print(paragraph.getText())
 
+def next_chapter_exists(soup):
+    """check if page has a next chapter button"""
```

Comment:
This comment style is actually called docstring
```suggestion
    """Check if page has a Next chapter button"""
```